portal/src/components/kinetic-form/widgets/signature.js

```diff
@@ -91,6 +151,10 @@ const SignatureComponent = forwardRef(
       setImageUrl('');
     };
 
+    const isSaveDisabled =
+      (activeTab === 'draw' && !imageUrl) ||
+      (activeTab === 'type' && !selectedStyle);
```

Comment:
The save should be disabled if the name field is empty on the `type` mode.

Also, I think we should have one of the font options selected by default so that the user doesn't need to select one if they don't care about the font choice, but verify this with Vivianne.